features/bookmarks/bookmark_create.feature

```diff
@@ -396,7 +396,7 @@ Scenario: Bookmark External Work link should be available to logged in users, bu
   Then I should not see "Bookmark External Work"
 
 @disable_caching
-Scenario: Editing a bookmark's tags should update the bookmark blurb
+Scenario: Editing a bookmark's tags should update the bookmark
```

Comment:
idk -- do we want to keep this test? It was actually written specifically to test that updating a tag would expire the cache on the bookmark blurb.